### DISCUSSION TITLE
TED Notice Search API Endpoint Issue - 404 Not Found Error

### DISCUSSION THREAD

**Discussion by Suraj-Dawkhare-Achilles** [OP-TED/eForms-SDK#816] [Q&A] [answered]
**Endpoint**: https://ted.europa.eu/api/v3.0/notices/search?fields=AA,AC,CY,DD,DI,DS,MA,NC,ND,OC,OJ,OL,OY,PC,PD,PR,RC,RN,RP,TD,TVH,TVL,TY,CONTENT&pageNum=1&pageSize=1&q=PD+%3E%3D+20240126+AND+PD+%3C%3D+20240128&reverseOrder=false&scope=2&sortField=PD

We have checked our implementation and verified that our requests are correctly formatted and were working before. We observe that API endpoint stopped working from **28th January 2024**. 

We are unsure if there have been any recent changes or if there's an issue on our end.

Could you please assist us in resolving this matter? If there have been any updates, changes to the API, or if we need to adjust our approach, we'd appreciate your guidance.

**Comment by bertrand-lorentz** [comment, marked as answer]
The TED website has switched to a new version on 28/01. See: https://ted.europa.eu/en/news/welcome-to-the-new-ted
As part of this, the search API has been changed, see the following for documentation: https://ted.europa.eu/en/simap/developers-corner-for-reusers##search-retrieve-data

This discussion forum is for the eForms SDK. For any further questions or issues related to the TED website, please contact the helpdesk: https://ted.europa.eu/en/contact

**Comment by Suraj-Dawkhare-Achilles** [comment]
Thank you for your direction, Can you please provide some link to understand the fields and other difference between old and new notice search API endpoint.

Thanks,
Suraj Dawkhare

________________________________
From: Bertrand Lorentz ***@***.***>
Sent: Friday, February 2, 2024 12:54
To: OP-TED/eForms-SDK ***@***.***>
Cc: Suraj Dawkhare ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [OP-TED/eForms-SDK] TED Notice Search API Endpoint Issue - 404 Not Found Error (Discussion #816)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

The TED website has switched to a new version on 28/01. See: https://ted.europa.eu/en/news/welcome-to-the-new-ted<https://ted.europa.eu/en/news/welcome-to-the-new-ted>
As part of this, the search API has been changed, see the following for documentation: https://ted.europa.eu/en/simap/developers-corner-for-reusers##search-retrieve-data<https://ted.europa.eu/en/simap/developers-corner-for-reusers##search-retrieve-data>

This discussion forum is for the eForms SDK. For any further questions or issues related to the TED website, please contact the helpdesk: https://ted.europa.eu/en/contact<https://ted.europa.eu/en/contact>

—
Reply to this email directly, view it on GitHub<https://github.com/OP-TED/eForms-SDK/discussions/816#discussioncomment-8346333>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BFZYGFPOPEIPTXVZJ34J54TYRTOYFAVCNFSM6AAAAABCWNWAVKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGNBWGMZTG>.
You are receiving this because you authored the thread.

*********************************************************************

Achilles Information Limited is registered in Scotland, registration Number SC137975 
and has its registered office at 7 Burnbank Business Centre, Souterhead Road, Altens, 
Aberdeen, AB12 3LF. 
 
This e-mail and any attachment are confidential and contain proprietary information. 
It is intended solely for the use of the individual or entity to which it is addressed. 
If you are not the intended recipient, please notify the author immediately by telephone 
or by replying to this e-mail and then delete all copies of the e- mail on your system. 
If you are not the intended recipient, you must not use, disclose, distribute, copy, 
print or rely on this e- mail.

Whilst we have taken reasonable precautions to ensure that this e-mail and any attachment 
has been checked for viruses, we cannot guarantee that they are virus free and we cannot 
accept liability for any damage sustained as a result of software viruses. We would advise 
that you carry out your own virus checks, especially before opening an attachment.


*********************************************************************

**Comment by bertrand-lorentz** [comment]
As indicated above, for any further questions or issues related to the TED website, please contact the helpdesk via https://ted.europa.eu/en/contact, or by e-mail at ted@publications.europa.eu

I'm closing this, as it is not related to the eForms SDK.
